readme.md

```diff
@@ -57,112 +63,117 @@ body {
 ```html
 <!-- out/index.html -->
 <p>Welcome</p>
-<style>body {
-	color: red;
-}</style>
+<style>
+  body {
+    color: red;
+  }
+</style>
 ```
 
-## `inline-images`
+### inline-images
 
 ```html
 <!-- src/index.html -->
 <p>Welcome</p>
-<img src="red_dot.png">
+<img src="red_dot.png" />
 ```
 
 `$ inline-images src/index.html out/index.html`
 
 ```html
 <!-- out/index.html -->
 <p>Welcome</p>
-<img src="data:image/png;base64, iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg==">
+<img
```

Comment:
Like above, the script doesn't line-wrap, and I think the output snippet should reflect the actual behavior.